spec/spec_helper.rb

```diff
@@ -17,7 +17,12 @@
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
   config.before(:each, type: :system) do
-    driven_by :selenium, using: :chrome, options: { args: ["headless", "disable-gpu", "no-sandbox", "disable-dev-shm-usage"] }
+    driven_by :selenium, using: :headless_chrome, options: { 
+      args: ["headless", "disable-gpu", "no-sandbox", "disable-dev-shm-usage"],
+      prefs: {
+        'download.default_directory' => "#{Rails.root}/tmp/capybara_downloads"
+      }
```

Comment:
What's the difference between spec_helper and rails_helper? Are they for different types of tests? Seems like selenium is being configured twice, but perhaps for different purposes?

Reply:
They're both files generated by RSpec, one is deliberately meant to be more customisable (the RailsHelper) and it holds more configuration as a result, also because for standard Rails classes like controller and unit tests. SpecHelper is a more generic helper that does not depend on Rails and can therefore be used for testing files in `lib` for example, which don't have access to the Rails environment unless specified. It has that extra config for Selenium because it's explicitly setting Selenium as the driver for system specs, whereas RailsHelper contains the general driver configuration.  

https://stackoverflow.com/questions/24145329/how-is-spec-rails-helper-rb-different-from-spec-spec-helper-rb-do-i-need-it